src/main/java/org/springframework/data/couchbase/repository/query/AbstractN1qlBasedQuery.java

```diff
@@ -122,10 +122,6 @@ protected static N1qlQuery buildQuery(Statement statement, JsonValue queryPlaceh
   protected Object executeDependingOnType(N1qlQuery query, N1qlQuery countQuery, QueryMethod queryMethod,
       Pageable pageable, Class<?> typeToRead) {
 
-    if (queryMethod.isModifyingQuery()) {
```

Comment:
should this check be more focused on checking for delete? i.e. we don't support updates yet, right?